contrib/offline/README.md

```diff
@@ -19,3 +23,21 @@ Step(2) can be operated with:
 ```shell
 manage-offline-container-images.sh   register
 ```
+
+## generate_list.sh
+
+This script generate download files list and images list by `roles/download/defaults/main.yml` file.
+
+Run this script will generate five files, all download files url in files.list, all images in images.list, all component version in version.sh.
```

Comment:
What are files.sh and images.sh in the five files?
According to the code, these files seem temporary files to generate the lists.

How about saying three files here and deleting these two files at the end of the script?

Reply:
Sorry, i forgot it.
Now, this script will generate three files, i combined `version.sh`, `files.sh` and `images.sh` into `generate.sh`